runtime/queries/javascript/highlights.scm

```diff
@@ -163,31 +163,32 @@
 [
   "as"
   "async"
-  "class"
-  "const"
   "debugger"
   "delete"
-  "export"
   "extends"
   "from"
   "function"
   "get"
-  "import"
   "in"
   "instanceof"
-  "let"
   "new"
   "of"
   "set"
   "static"
   "target"
   "try"
   "typeof"
-  "var"
   "void"
   "with"
 ] @keyword
 
+[
+  "class"
+  "let"
+  "const"
+  "var"
+] @keyword.storage.type
```

Comment:
`const` should be `keyword.storage.modifier`

Reply:
Not in javascript/typescript. It's used in place of let, var etc. It's a variable that cannot be reassigned.

```ts
/** can be re-assigned but not re-declared */
let someVar = 'hello world';
someVar = 'new string'; //OK!
let someVar = 'goodbye world' //ERROR!

/** can be re-assigned and re-declared */
var someVar = 'hello world;
someVar = 'new string'; //OK!
var someVar = 'goodbye world'; //OK!

/** cannot be re-assigned or re-declared */
const someVar = 'hello world';
someVar = 'new string'; //ERROR!
const someVar = 'goodbye world'; //ERROR!
```
There's a little more nuance to the differences between var and let but that should outline it.

There is another use for it in typescript when doing type assertions:

```ts
let someVar = someOtherVar as const
```

but I'm not familiar enough with the query syntax to approach that.